lightkurve/periodogram.py

```diff
@@ -321,6 +322,48 @@ def bin(self, binsize=10, method='mean'):
         binned_pg.power = binned_power
         return binned_pg
 
+    def smooth(self, filter_width = 0.1/u.day):
+        """Smooths the power spectrum using a simple one dimensional smoothing
+        filter. This method requires a Periodogram class built using an evenly
+        spaced grid of periods.
```

Comment:
Nitpicking: in the docstring, it is worth saying out loud exactly what operation is being applied, i.e. `convolving with a numpy Box1DKernel`, because many users will probably understand what this means.

Reply:
Done!